pages/docs/index.md

```diff
@@ -62,63 +124,257 @@ curl -X GET "https://<ENVIRONMENT_URL>/v1/user?id=PvRoVxxiJ2gul3qLwKOVaJ0HCNs1"
+- **`height_feet`** (float, Optional)
 
-- **`date_of_birth`** (isodate): 
-  - **Description**: User's date of birth.
+- **`height_inches`** (float, Optional)
 
-- **`allow_trume_login`** (boolean, Optional): 
-  - **Description**: If true, dispatches an email for TruMe password creation.
+- **`weight`** (float, Optional)
+
+- **`overall_health`** (string, Optional):
+	- **Description**: ['Fair', 'Excellent', 'Good', 'Poor']
+
+- **`mental_health`** (string, Optional):
+	- **Description**: ['Fair', 'Excellent', 'Good', 'Poor']
+
+- **`social_life`** (string, Optional):
+	- **Description**: ['Yes', 'No', "I'd rather not say"]
+
+- **`stress`** (string, Optional):
+	- **Description**: ['A little stressed', 'Somewhat stressed', 'Really stressed', 'Not at all', 'Extremely stressed']
+
+- **`has_cancer`** (string, Optional):
+	- **Description**: ['Yes', 'No', "I'd rather not say"]
+
+- **`has_diabetes`** (string, Optional):
+	- **Description**: ['Yes', 'No', "I'd rather not say"]
+
+- **`medicine_consumption`** (string, Optional):
+	- **Description**: ['Yes', 'No']
+
+- **`supplement_consumption`** (string, Optional):
+	- **Description**: e.g: Vitamin D
+
+- **`coffee_consumption`** (string, Optional):
+	- **Description**: ['Yes', 'No']
+
+- **`tobacco_consumption`** (string, Optional):
+	- **Description**: ['Yes', 'No']
+
+- **`waking_up_condition`** (string, Optional):
+	- **Description**: ['Yes', 'No']
+
+- **`trouble_sleeping`** (string, Optional):
+	- **Description**: ['Yes', 'No']
+
+- **`alcohol_consumption`** (string, Optional):
+	- **Description**: ['0', '1-2', '3-5', '6-9', '+10']
+
+- **`sleep_hours`** (string, Optional):
+	- **Description**: ['<5', '6', '7', '8', '9', '>10']
 
+- **`physical_activity_days`** (string, Optional):
+	- **Description**: ['0', '1', '2', '3', '4', '5', '+6']
+
+- **`registered_at`** (datetime, Optional):
+	- **Description**: e.g: 2024-11-27T10:00:00
 
 #### Example Request
 
 ```bash
-curl -X GET "https://<ENVIRONMENT_URL>/v1/user" \
--H "App-Key: YOUR_TOKEN"
+curl --request POST \
+  --url 'http://<ENVIRONMENT_URL>/v1/kits' \
+  --header 'App-Key: YOUR-TOKEN' \
+  --header 'Content-Type: application/json' \
+  --data '{
+	"kit_number": "TEST1234",
+	"user_id": "vd7KETc6UHuGbeMRv9EnFdEh0mfD",
+	"height_feet": 5.0,
+	"height_inches": 70.0,
+	"weight": 160.0,
+	"overall_health": "Good",
+  "mental_health": "Fair",
+	"social_life": "I'\''d rather not say",
+	"stress": "Not at all",
+	"has_cancer": "No",
+	"has_diabetes": "Yes",
+	"medicine_consumption": "Yes",
+	"supplement_consumption": "Vitamin D",
+	"coffee_consumption": "Yes",
+	"tobacco_consumption": "No",
+	"waking_up_condition": "Yes",
+	"trouble_sleeping": "No",
+	"alcohol_consumption": "0",
+	"sleep_hours": "8",
+	"physical_activity_days": "3",
+	"registered_at": "2024-11-27T10:00:00"
+}'
 ```
 
 #### Response
+Empty
 
-```json
-{
-	"user_id": "PvRoVxxiJ2gul3qLwKOVaJ0HCNs1"
-}
+&nbsp;
+### `PATCH /v1/kits/{barcode}` (Only staging yet)
+
+Update an kit.
```

Comment:
I think devs need a bit of context here. "Kits" might be something they only associate with the kit itself, and not the patient data. It can be a bit confusing for them. Maybe we can rethink of naming?